### PR TITLE
Update REST API interfaces 

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

**Comment by janinko** [comment]
The intenal endpoints (Build Task, BPM and Debug) are not finished, as there are missing DTOs and there is probably need some more serious changes (at least in the BPM)

File: rest-api/src/main/java/org/jboss/pnc/rest/api/parameters/GroupBuildParameters.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Parameter(description = "Should we force the rebuild of all build configurations?")
+    @QueryParam("forceRebuild")
+    @DefaultValue("false")
+    boolean forceRebuild;
+
+    @Parameter(description = "Should we add a timestamp during the alignment? Valid only for temporary builds.")
+    @QueryParam("timestampAlignment")
+    @DefaultValue("false")
+    boolean timestampAlignmen;
```
Comment (matedo1) [inline rest-api/src/main/java/org/jboss/pnc/rest/api/parameters/GroupBuildParameters.java]: Typo, correct: "timestampAlignment"

File: rest-api/src/main/java/org/jboss/pnc/rest/api/parameters/BuildParameters.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    @Parameter(description = "Should we keep the build container running, if the build fails?")
+    @QueryParam("keepPodOnFailure")
+    @DefaultValue("false")
+    boolean keepPodOnFailure;
+
+    @Parameter(description = "Should we add a timestamp during the alignment? Valid only for temporary builds.")
+    @QueryParam("timestampAlignment")
+    @DefaultValue("false")
+    boolean timestampAlignmen;
```
Comment (matedo1) [inline rest-api/src/main/java/org/jboss/pnc/rest/api/parameters/BuildParameters.java]: Typo, correct: "timestampAlignment"

**Comment by janinko** [comment]
Here is the openapi.json file https://gist.github.com/janinko/860170807ff17e200664dcbdb6fd1391

**Pull request merged**